packages/manager/modules/manager-components/src/resource-selector/template.html

```diff
@@ -0,0 +1,11 @@
+<div class="oui-tile d-flex m-2">
+    <h6 class="w-50" data-ng-bind="$ctrl.title"></h6>
+    <div class="text-right ml-auto mr-3" ng-transclude="price">
+        <h6 class="m-0">
```

Comment:
Isn't the `h6` overridden with the transclude here ? 
Also maybe we should just use the style as we don't know if other headings will be provided across the same view

Reply:
Yeah... The user can override it using the resourceSelectorPrice. The user can directly give the price string or transclude the required content using `resourceSelectorPrice`. `resourceSelectorPrice` takes higher precedence